keras/preprocessing/image.py

```diff
@@ -358,6 +358,19 @@ def img_to_array(img, data_format=None):
     return x
 
 
+def save_img(path, x, data_format=None, scale=True):
+    """Save an image stored as a Numpy Array to a path or file object.
+
+    # Arguments
+        path: Path or file object.
+        x: Numpy array.
+        data_format: Image data format.
```

Comment:
Explicitly mention that this is 'channels_first'/'channels_last' (since it may be confusing vs. image file format).

Reply:
I updated the docstring for the `data_format` parameter to explicitly state the options of `{"channels_first" or "channels_last"}`.